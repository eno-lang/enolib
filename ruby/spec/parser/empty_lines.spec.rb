input = "\n" +
        " \n" +
        "  \n" +
        "   \n" +
        "\n" +
        " \n" +
        "  \n" +
        "   \n"

describe Enolib::Parser do
  describe 'Empty lines' do
    context = Enolib::Context.new(input)

    it 'parses as expected' do
      expect(context.document).to match_snapshot
    end
  end
end
