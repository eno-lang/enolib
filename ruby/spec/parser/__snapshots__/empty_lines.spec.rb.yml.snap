---
"./spec/tokenizer/empty_lines.spec.rb[1:1:1]":
- :index: 0
  :line: 0
  :type: :empty_line
  :length: 0
- :index: 1
  :line: 1
  :type: :empty_line
  :length: 1
- :index: 3
  :line: 2
  :type: :empty_line
  :length: 2
- :index: 6
  :line: 3
  :type: :empty_line
  :length: 3
- :index: 10
  :line: 4
  :type: :empty_line
  :length: 0
- :index: 11
  :line: 5
  :type: :empty_line
  :length: 1
- :index: 13
  :line: 6
  :type: :empty_line
  :length: 2
- :index: 16
  :line: 7
  :type: :empty_line
  :length: 3
- :index: 20
  :length: 0
  :line: 8
  :type: :empty_line
"./spec/parser/empty_lines.spec.rb[1:1:1]":
  :elements: []
  :type: :document
